test/JDBC/jdbc_schedule

```diff
@@ -8,6 +8,8 @@
 #    new line
 # 6. If you want the framework to not run certain files, use: ignore#!#<test name>
 
+# check babelfish version first
+cmd#!#postgresql#!#select cast(case when cast(sys.SERVERPROPERTY('BabelfishVersion') as varchar(20)) LIKE '_._._' THEN 'valid' else 'invalid' end as sys.varchar(20));
```

Comment:
why would we do this here in schedule file?

Reply:
To repro this issue, We need to invoke given function right after installing babelfish. So, I could not find any other better place to verify that SERVERPROPERTY('BabelfishVersion') works as intended (at least does not crash or raise an error)